tests/integration/kubernetes/tests_common.sh

```diff
@@ -112,18 +121,15 @@ exec_host() {
 	# [bats-exec-test:38] INFO: k8s configured to use runtimeclass
 	# bash: line 1: $'\r': command not found
 	# ```
-	output="$(kubectl debug -qi "node/${node}" --image=quay.io/bedrock/ubuntu:latest -- chroot /host bash -c "${command}" | tr -d '\r')"
+	local command_output
+	command_output=$(kubectl exec -q "${new_debugger_pod}" -- chroot /host bash -c "${command}" | tr -d '\r' 2>/dev/null)
+	local exit_code=$?
```

Comment:
Have we verified that `kubectl exec` returns the underlying exit code?

Reply:
yeah, we could preserve the logic using `command+='; echo -en \\n$?'` for this. thanks.